content/building-blocks/analyze-data/non-parametric-tests/introduction.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Intro to non parametric tests"
+description: "Describing the types of non parametric tests and their usage"
+keywords: "non, para, test, binom, chi, wilcoxon, mcnemar, fisher, median, mann, whitney, kruskal, wallis"
+weight: 1
+date:
+draft: false
+aliases:
+  - /non-para-intro
+---
+
+## Overview
+
+Performing statistical tests like t-test and f-test are imperative in any empirical study to help test the hypotheses about population parameters. However, parametric tests like these make certain assumptions about the underlying population distribution which are not always reasonable to make. Hence, in such a case, one should use a non-parametric test instead.
```

Comment:
This is going in a good direction. Can you try to more explicitly state WHY we need these tests? The language you're using is a bit too complex for students to understand. Working with an example in your first paragraph will help.